### DISCUSSION TITLE
Brainstorming Elemente für Frontend und Backend Layout

### DISCUSSION THREAD

**Discussion by thelf** [Webmasters-Europe/codemeetup-forum#17] [Ideas] [answered]
Hallo zusammen,

ich habe mir ein paar Gedanken über die Elemente gemacht die sichtbar im Backend und Frontend zu sehen seien werden. Es geht noch gar nicht um die Gestaltung, sondern einfach welche Elemente notwendig und sinnvoll sind.

Ich fange einfach mal eine Liste an die ihr gerne ergänzen könnt.

**Frontend:**
- Header mit Titel
- Navigation
- Breadcrumb Navigation
- Suchfeld
- Login Bereich mit Input Feldern und Button
- Schaltflächen oder Links für Registrierung und Passwort vergessen
- Abmelde Button 
- Textfelder um Beiträge zu erstellen
- Absenden Button um Nachricht in das Forum zu stellen
- Checkbox (Bei Antworten per E-Mail benachrichtigen)
- Box(en) für den Newsbereich
- Footer mit Anmeldung (Impressum, Datenschutz usw.)

**Comment by Cosnavel** [comment, marked as answer]
Willst du die Komponenten alle selbst designen oder eine Bibliothek verwenden?

**Reply by thelf** [reply]
Unsere Idee war aus Bootstrap die Elemente zu verwenden da wir dann schon ein Grunddesign haben. Das können wir ja nach Belieben anpassen wie wir das möchten, aber so haben haben wir schonmal Komponenten die wir für einen Prototypen verwenden können und auch parallel an JavaScript arbeiten können.
